ui/README.md

```diff
@@ -1,15 +1,15 @@
+**Table of Contents**
```

Comment:
Gets deleted by running `yarn doctoc README.md` unless outside the comment block